File: Modul 3/C/PRAK305.c
#include <stdio.h.>

int main (){
    int input_detik;
    printf("Input\n");
    scanf("%d", &input_detik);
    printf("Output\n");

    int hari = input_detik / (3600*24);
    int sisa_hari = input_detik % (3600*24);
    int jam = sisa_hari / 3600;
    int sisa_jam = input_detik % 3600;
    int detik = sisa_jam % 60;
    int menit = sisa_jam / 60;

    if (input_detik < 60) {
        printf("00:00:%02d", detik);
    } else if (input_detik > 60 && input_detik < 3600) {
        printf("00:%02d:%02d", menit, detik);
    } else if (input_detik >= 3600 && input_detik < 86400) {
        printf("%02d:%02d:%02d", jam, menit, detik);
    } else if (input_detik > 86400) {
        printf("%d hari ", hari);
        printf("%02d:%02d:%02d", jam, menit, detik);
    }
return 0;
}

File: Modul 3/Python/PRAK305.py
print("Input")
input_detik = int(input())
hari = input_detik // (3600*24)
sisa_hari = input_detik % (3600*24)
jam = sisa_hari // 3600
sisa_jam = sisa_hari % 3600
detik = sisa_jam % 60
menit = sisa_jam // 60

print("Output")
if (input_detik < 60):
    print(f"00:00:{detik:02}")
elif (input_detik > 60) and (input_detik < 3600): 
    print(f"00:{menit:02}:{detik:02}")
elif (input_detik >= 3600) and  (input_detik < 86400):
    print(f"{jam:02}:{menit:02}:{detik:02}")
else: 
    print(f"{hari:01} hari {jam:02}:{menit:02}:{detik:02}")